first_order_model/shrink_util.py

```diff
@@ -0,0 +1,1266 @@
+            index += 1
+
+    # Build graph
+    if os.environ.get('CONV_TYPE', 'regular') == 'regular':
+        add_names([
+            'dense_motion_network.hourglass.encoder.down_blocks.0.conv',
+            'dense_motion_network.hourglass.encoder.down_blocks.0.norm',
+            'dense_motion_network.hourglass.encoder.down_blocks.1.conv',
+            'dense_motion_network.hourglass.encoder.down_blocks.1.norm',
+            'dense_motion_network.hourglass.encoder.down_blocks.2.conv',
+            'dense_motion_network.hourglass.encoder.down_blocks.2.norm',
+            'dense_motion_network.hourglass.encoder.down_blocks.3.conv',
+            'dense_motion_network.hourglass.encoder.down_blocks.3.norm',
+            'dense_motion_network.hourglass.encoder.down_blocks.4.conv',
+            'dense_motion_network.hourglass.encoder.down_blocks.4.norm',
+            'dense_motion_network.hourglass.decoder.up_blocks.0.conv',
+            'dense_motion_network.hourglass.decoder.up_blocks.0.norm',
+            'dense_motion_network.hourglass.decoder.up_blocks.1.conv',
+            'dense_motion_network.hourglass.decoder.up_blocks.1.norm',
+            'dense_motion_network.hourglass.decoder.up_blocks.2.conv',
+            'dense_motion_network.hourglass.decoder.up_blocks.2.norm',
+            'dense_motion_network.hourglass.decoder.up_blocks.3.conv',
+            'dense_motion_network.hourglass.decoder.up_blocks.3.norm',
+            'dense_motion_network.hourglass.decoder.up_blocks.4.conv',
+            'dense_motion_network.hourglass.decoder.up_blocks.4.norm'
+        ])
+
+        # Add the dense motion skip connections
+        add('dense_motion_network.hourglass.encoder.down_blocks.3.norm',
+            'dense_motion_network.hourglass.decoder.up_blocks.1.conv')
+        add('dense_motion_network.hourglass.encoder.down_blocks.2.norm',
+            'dense_motion_network.hourglass.decoder.up_blocks.2.conv')
+        add('dense_motion_network.hourglass.encoder.down_blocks.1.norm',
+            'dense_motion_network.hourglass.decoder.up_blocks.3.conv')
+        add('dense_motion_network.hourglass.encoder.down_blocks.0.norm',
+            'dense_motion_network.hourglass.decoder.up_blocks.4.conv')
+
+        # Add the dense motion outputs partly (First part)
+        add('dense_motion_network.hourglass.decoder.up_blocks.4.norm',
+            'dense_motion_network.mask')
+        add('dense_motion_network.hourglass.decoder.up_blocks.4.norm',
+            'dense_motion_network.occlusion')
+        add('dense_motion_network.hourglass.decoder.up_blocks.4.norm',
+            'dense_motion_network.lr_occlusion')
+        add('dense_motion_network.hourglass.decoder.up_blocks.4.norm',
+            'dense_motion_network.hr_background_occlusion')
+
+        add('lr_first.conv', 'lr_first.norm')
+        add_names([
+            'hr_first.conv', 'hr_first.norm', 'hr_down_blocks.0.conv',
+            'hr_down_blocks.0.norm'
+        ])
+        add_names([
```

Comment:
Please try to clean up these names by using for loops and pattern matching rather than repeating it so many times.